cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt

```diff
@@ -161,13 +188,37 @@ datasource "ds_get_anomalies" do
 end
 
 datasource "ds_filter_anomalies" do
-  run_script $js_filter_anomalies, $ds_get_anomalies, $param_min_spend, rs_org_id, $param_days, $param_dimensions, $param_cost_metric, $param_anomaly_limit
+  run_script $js_filter_anomalies, $ds_get_anomalies, $param_min_spend, rs_org_id, $param_days, $param_dimensions, $ds_rbd_dimensions, $param_cost_metric, $param_anomaly_limit
 end
 
 ###############################################################################
 # Scripts
 ###############################################################################
 
+script "js_rbd_dimensions", type: "javascript" do
+  parameters "param_rbd_dimensions", "ds_rule_based_dimension"
+  result "cleaned_rbd_dimensions"
+  code <<-EOS
+    var cleaned_rbd_dimensions = {
+      rbd_dimensions_ids: [],
+      messages: []
+    };
+    var exist = false;
+    _.each(param_rbd_dimensions, function(dimension) {
+      exist = false;
+      _.each(ds_rule_based_dimension, function(rbd) {
+        if (dimension === rbd.name || dimension === rbd.id) {
+          cleaned_rbd_dimensions.rbd_dimensions_ids.push(rbd.id);
+          exist = true;
+        }
+      });
+      if (!exist) {
```

Comment:
In this point Idk if "exist" or "exists" variable is necessary, maybe you can just validate if element is in array, something like:

`if (!cleaned_rbd_dimensions.rbd_dimensions_ids.includes(dimension))`

But that's only an idea, maybe will be better if we use that "exists" variable for performance reasons.